config/initializers/training_routes.rb

```diff
@@ -47,3 +47,10 @@
   school_direct_salaried: "QTS",
   school_direct_tuition_fee: "QTS",
 }.freeze
+
+EARLY_YEARS_ROUTES = %i[
```

Comment:
These were modified a couple of days ago to match their new names before any data went in. See `TRAINING_ROUTE_ENUMS`

Reply:
Also feels like we don't need `TRAINING_ROUTE_AWARD_TYPE` anymore? Feels strange to have them both in here as constants.